.github/CONTRIBUTING.md

```diff
@@ -144,6 +144,12 @@ make html
 
 and open `docs/build/html/index.html` in your browser.
 
+Notes:
+ - you need to have installed latex for rendering math equations. To do so you install TeXLive, you can do one of following:
+    * on linux run `apt install texlive`
+    * use [RTD docker image](https://hub.docker.com/r/readthedocs/build)
+- with PL used class meta you need to use python 3.7 or higher
```

Comment:
is it related only to docs? if not, maybe mention it elsewhere?

Reply:
to clarify I mean the 2nd bullet point "with PL used class meta you need to use python 3.7 or higher"

Reply:
I found it important only for docs as all other test are passing also for 3.6